e2e/suites/actions/copy.test.ts

```diff
@@ -127,18 +132,18 @@ describe('Copy content', () => {
     await apis.user.nodes.createFolder(folderSiteFav, docLibId);
     await apis.user.nodes.createFolder(folderSiteSearch, docLibId);
 
-    await apis.user.shared.waitForApi({ expect: 4 });
-    await apis.user.favorites.waitForApi({ expect: 7 });
+    await apis.user.shared.waitForApi({ expect: 5 });
+    await apis.user.favorites.waitForApi({ expect: 8 });
 
     await loginPage.loginWith(username);
     done();
   });
 
-  afterAll(async (done) => {
-    await apis.user.nodes.deleteNodeById(sourceId);
-    await apis.user.sites.deleteSite(siteName);
-    done();
-  });
+  // afterAll(async (done) => {
```

Comment:
Can you please remove this code if it is not needed anymore?

Reply:
it's needed! :)
it's probably commented out cause this is still WIP I assume